udata/tests/api/test_user_api.py

```diff
@@ -1,9 +1,11 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+import json
```

Comment:
No need to manually parse `response.data` every time, you can directly use `response.json`